community/people/CGK/index.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: person
+title: Carena
```

Comment:
I just noticed that you may want your fill name in this since it is used to add text to the page:

![image](https://user-images.githubusercontent.com/764326/122239299-704d8480-ce86-11eb-98b6-d8c504453361.png)